proposal/chrisPyr/README.rst

```diff
@@ -0,0 +1,74 @@
+================
+NSD Term Project
+================
+--------------------------
+chrisPyr @ 2021 NSD autumn
+--------------------------
+
+Basic Information
+=================
+| Project GitHub Link:
+| https://github.com/chrisPyr/My_Universe
+
+Problem to Solve
+================
+
+| We all learned "Kepler's laws of planetary motion"[1] in high school.
+| The Kepler's laws can be explained and proved by Newtonian Mechanics[2].
+| However, in fact, the laws would fail to predict the orbits or speed of planets accurately in some certain conditions, e.g. Perihelion Motion of Mercury[3](fig. 1) and that's why we need "Theory of Relativity" to help us to describe and predict more precisely.
+| Einstein did prove this on his paper[4].
+| I plan to build up a program that can predict the orbits of planets but only take Newtonian Mechanics into account.
+| The final product should be able to show the orbits, periods of planets. And it can be compared with the real world data to prove the correctness and also incorrectness of Newtonian Mechanics.
+| And also do I want to add some intersting feature into it, e.g. users can add/delete planets into/in the system to see how the entire system changes.
+| If I can finish all the above features on time. Then the next step is to also take Theory of Relativity into account(and that would be much harder).
+
+.. image:: images/Precessing_Kepler_orbit_280frames_e0.6_smaller.gif
+   
+
+Prospective Users
+=================
+
+| This can be used as a supplementary tutorial to high school student.
+
+System Architecture
+===================
+
+| In my plan, the default of building such system would create the solar system in a gif or mp4 format.
+
+Program Features:
+    - export information of calculated result such as formula of orbit, speed/period of a planet
+    - export a gif or mp4 file to show the drawing result
+    - add user-defined planets
+    - delete planets
+
+Constraints:
+    - Newtonian Mechanics may not describe real world precisely
```

Comment:
It's actually quite precise.  It works on most problems accessible to human flesh.